### ISSUE TITLE
Caffe seemed to be paused while training dataset?

### ISSUE THREAD

**Issue by grantfee** [NVIDIA/DIGITS#2018]
I'm using nvidia DIGITS to train image dataset count about 20000. But caffe could not finish the training process. Any help? caffe output log shows as below and no error output

I0410 12:36:02.017575 19042 net.cpp:264] This network produces output loss2/accuracy-top5
I0410 12:36:02.017578 19042 net.cpp:264] This network produces output loss2/loss
I0410 12:36:02.017724 19042 net.cpp:284] Network initialization done.
I0410 12:36:02.018319 19042 solver.cpp:60] Solver scaffolding done.
I0410 12:36:02.018541 19042 caffe.cpp:231] Starting Optimization
I0410 12:36:02.018546 19042 solver.cpp:304] Solving
I0410 12:36:02.018549 19042 solver.cpp:305] Learning Rate Policy: step
I0410 12:36:02.032096 19042 solver.cpp:362] Iteration 0, Testing net (#0)
I0410 12:36:02.032110 19042 net.cpp:723] Ignoring source layer train-data
I0410 12:36:02.032114 19042 net.cpp:723] Ignoring source layer label_train-data_1_split
